README.md

```diff
@@ -1,49 +1,185 @@
 # iit-backend
 
-Server side code for the International Immunity Tracker
-
+Server side code for the International Immunity Tracker.
+
+# Table of Contents
+
+- [iit-backend](#iit-backend)
+- [Table of Contents](#table-of-contents)
+- [Set up](#set-up)
+  - [Cloning](#cloning)
+  - [Terminal-based installation](#terminal-based-installation)
+    - [Dependencies](#dependencies)
+  - [PyCharm on MacOS](#pycharm-on-macos)
+    - [Required Software](#required-software)
+    - [Project Configuration](#project-configuration)
+      - [Creating Virtual Environment](#creating-virtual-environment)
+      - [Installing Required Packages](#installing-required-packages)
+      - [Run Configuration](#run-configuration)
+  - [PyCharm on Windows](#pycharm-on-windows)
+    - [Required Software](#required-software-1)
+    - [Project Configuration](#project-configuration-1)
+      - [Creating virtual environment](#creating-virtual-environment-1)
+      - [Installing Required Packages](#installing-required-packages-1)
+      - [Run Configuration](#run-configuration-1)
+  - [Postgres](#postgres)
+  - [Changing Alembic to use Flask SQLAlchemy](#changing-alembic-to-use-flask-sqlalchemy)
+  - [Delete Previous Tables & Migrations](#delete-previous-tables--migrations)
+  - [Upgrade to Using Newest Migrations](#upgrade-to-using-newest-migrations)
+    - [Run migrations](#run-migrations)
+    - [Populate local database](#populate-local-database)
+  - [Running test suite](#running-test-suite)
+  - [Loading Tableau CSV to Google Sheets](#loading-tableau-csv-to-google-sheets)
+- [Helpful Code Snippets](#helpful-code-snippets)
+  - [Working With Postgres](#working-with-postgres)
+  - [Running ETL and Local Server](#running-etl-and-local-server)
+  - [Export and Import](#export-and-import)
+  - [Running Dockerized Flask App](#running-dockerized-flask-app)
+- [Infrastructure Documentation (Current - Vanilla EC2)](#infrastructure-documentation-current---vanilla-ec2)
+  - [CI/CD](#cicd)
+    - [Continuous Integration](#continuous-integration)
+    - [Continuous Deployment](#continuous-deployment)
+    - [Results](#results)
+  - [Cronjobs](#cronjobs)
+  - [`tmux` sessions](#tmux-sessions)
+- [Infrastructure Documentation (Future - Elastic Beanstalk)](#infrastructure-documentation-future---elastic-beanstalk)
 # Set up
 
 ## Cloning
 
 `git clone https://github.com/serotracker/iit-backend.git`
```

Comment:
make this an action: run <git clone> in your terminal

Reply:
Let's reformat this section:

Make the following hierarchy:

- Set up
-- Cloning
-- Environment configuration
--- Using the terminal
--- Using PyCharm
---- MacOsS
---- Windows

Does that make sense?
Because cloning is common to all users regardless of OS type, they just run git clone in their terminal. You can mention that little snippet about if using WSL clone it into your home directory. Then next there are 3 ways to set up the environment: terminal, Pycharm MacOS, PyCharm Windows. then within each of these three options we need to include the following steps: 1) install pip 2) create your virtual env 3) create .env and load it into environment 4) install requirements.txt 5) how to run the actual script.

does this make sense? think it would just make it a bit more organized and easier to follow. you have all the content here just needs a bit of reorganizing.

Reply:
Yeah I didn't like how it was laid out but for some reason couldn't find a better way. This is much better.